src/main/java/seedu/duke/storage/DoctorAppointmentStorage.java

```diff
@@ -0,0 +1,56 @@
+package seedu.duke.storage;
+
+import seedu.doctorappointments.AppointmentActions;
+import seedu.doctorappointments.DoctorAppointment;
+import seedu.drugs.Drug;
+
+import javax.print.Doc;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class DoctorAppointmentStorage {
+    private final String filePath;
+    private final File file;
+
+    public DoctorAppointmentStorage(String filePath) {
+        this.filePath = filePath;
+        this.file = new File(filePath);
+    }
+
+    public void createFile() throws IOException {
+        Path pathToFile = Paths.get(filePath);
+        Files.createDirectories(pathToFile.getParent());
+        file.createNewFile();
+    }
+
+    public AppointmentActions loadFile() throws FileNotFoundException {
```

Comment:
Whole storage component looks good to me and seems to meet basic functionality. Might want to consider adding data validation in future iterations.